hoodie-common/src/main/java/com/uber/hoodie/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -97,7 +97,7 @@ protected HoodieActiveTimeline(FileSystem fs, String metaPath, String[] included
     public HoodieActiveTimeline(FileSystem fs, String metaPath) {
         this(fs, metaPath,
             new String[] {COMMIT_EXTENSION, INFLIGHT_COMMIT_EXTENSION, DELTA_COMMIT_EXTENSION,
-                INFLIGHT_DELTA_COMMIT_EXTENSION, COMPACTION_EXTENSION,
+                INFLIGHT_DELTA_COMMIT_EXTENSION, COMPACTION_EXTENSION, INFLIGHT_ROLLBACK_EXTENSION, ROLLBACK_EXTENSION,
```

Comment:
@n3nash is this strictly needed for this PR?  if not can you please revert the additional extensions .. for eg, I dont even think we write a .inflight for rollbacks.. so probably better to fix this in a separate PR if this is not needed here